### PR TITLE
Use openshift-e2e-vsphere workflow for cluster-image-registry-operator tests

### PR DESCRIPTION
**Pull request by dmage**
As vSphere moved toward the steps workflow, this PR updates cluster-image-registry-operator vsphere jobs to use steps.

### CODE REVIEW

**Comment by dmage** [comment]
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/11743/rehearse-11743-pull-ci-openshift-cluster-image-registry-operator-master-e2e-vsphere-operator/1306192092912947200

at least TestImageRegistryRemovedWithImages is expected to fail, other failures need to be investigated

but the goal of this PR is just to start tests, currently the job [cannot setup the cluster](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-image-registry-operator/608/pull-ci-openshift-cluster-image-registry-operator-master-e2e-vsphere-operator/1306190862400622592)

**Comment by dmage** [comment]
@jcpowermac all 3 e2e-vsphere jobs are failed, can we consider it as a success? :)

**Comment by jcpowermac** [comment]
> @jcpowermac all 3 e2e-vsphere jobs are failed, can we consider it as a success? :)

4.6 currently does have issues. I would say this is lgtm from me but expect failures until 4.6 on vsphere is more stable (hopefully soon)

File: ci-operator/config/openshift/cluster-image-registry-operator/openshift-cluster-image-registry-operator-master.yaml [hunk 1]
```diff
@@ -78,14 +78,22 @@ tests:
...
     cluster_profile: vsphere
+    test:
+    - artifact_dir: /tmp/artifacts
+      as: e2e-test
+      commands: JUNIT_REPORT=1 make test-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
```
Comment (ricardomaraschini) [inline ci-operator/config/openshift/cluster-image-registry-operator/openshift-cluster-image-registry-operator-master.yaml:95]: is this enough memory for our e2e tests?

**Review by ricardomaraschini — APPROVED** [review]
/lgtm

I just let a comment but it looks good otherwise.

**Pull request merged**